packages/mom5/package.py

```diff
@@ -533,53 +592,50 @@ def edit(self, spec, prefix):
 	hpm -r -o $*.hpm $*.x
 """
 
-        fullconfig = config[self.compiler.name] + config["post"]
+        fullconfig = config[pkg.compiler.name] + config["post"]
         print(fullconfig)
         with open(makeinc_path, "w") as makeinc:
             makeinc.write(fullconfig)
 
-    def build(self, spec, prefix):
+    def build(self, pkg, spec, prefix):
 
         # cd ${ACCESS_OM_DIR}/src/mom/exp
         # export mom_type=ACCESS-OM
         # ./MOM_compile.csh --type $mom_type --platform spack
-        with working_dir(join_path(self.stage.source_path, "exp")):
+        with working_dir(join_path(pkg.stage.source_path, "exp")):
             build = Executable("./MOM_compile.csh")
 
-            if self.spec.satisfies("+restart_repro"):
+            if pkg.spec.satisfies("+restart_repro"):
                 build.add_default_env("REPRO", "true")
                 print("INFO: +restart_repro applied")
 
-            if self.__builds[self.__version]["gtracers"]:
-                build.add_default_env("SPACK_GTRACERS_EXTERNAL", "true")
```

Comment:
This is removed because the plan is not to have generic tracers support in the Makefile builds.